docs/architecture.rst

```diff
@@ -0,0 +1,189 @@
+ How ppb Fits Together
+===========================================================
+
+This document is an overall reference of what ppb is, how it's organized, and
+how to use the various pieces.
+
+The Conceptual Necessities
+-----------------------------------------------------------
+.. _arch-the-basics:
+
+Put some introduction here.
```

Comment:
I might suggest including something like:
"Games developed using PPB are generally made of Sprites organized into a hierarchy of Scenes.  These Sprites behave according to rules laid out in Systems, and communicate via Events.  Sprites load Assets from storage.  A special GameObject manages Time and another handles the virtual Camera."

Reply:
In thinking about it, a paragraph like this might mean the list I suggested above is not needed.